### PR TITLE
remove the indie-mark-col field in row change event

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
remove the indie-mark-col field in row change event

### CODE REVIEW

File: cdc/sink/dispatcher/interface.go [hunk 1]
```diff
@@ -103,13 +103,18 @@ func NewDispatcher(cfg *config.ReplicaConfig, partitionNum int32) (Dispatcher, e
 		rule.fromString(ruleConfig.Dispatcher)
 		switch rule {
 		case dispatchRuleRowID, dispatchRuleIndexValue:
+			if cfg.EnableOldValue {
+				log.Warn("This index-value distribution mode " +
+					"does not guarantee row-level orderliness when " +
+					"switching on the old value, so please use caution!")
+			}
```
Comment (overvenus) [inline cdc/sink/dispatcher/interface.go:110]: Could you add the warning to cli too?

Comment (leoppro) [reply cdc/sink/dispatcher/interface.go:110]: fine

**Pull request merged**